src/APP/Infrastructure/adapters/repositories/PrismaRepositories/UserRepository.ts

```
@@ -0,0 +1,151 @@
+        } catch (error: any) {
+            this.logger.error(error.message);
+            return Result.Err(new DbMalfunction("update"));
+        }
+    }
+
+    async deleteById(userId: UUIDVo): Promise<RepositoryResult<User>> {
+        try {
+            const removedUser = await this.client.user.delete({
+                where: { userId: userId.serialize() }
+            });
+            return removedUser ? Result.Ok(User.fromObj(UserDTO.toDomain(removedUser))) : Result.Err(new UserNotFound(userId.serialize()));;
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)